src/main_app/admin/flask_admin_panel.py

```
@@ -0,0 +1,87 @@
+    OwidSlugRedirectRecord,
+    SettingRecord,
+    TemplateRecord,
+    UserRecord,
+)
+
+
+class WrapModelView(ModelView):
+    ignore_hidden = True
+    form_excluded_columns = ("created_at", "updated_at", "token")
+    column_display_actions: bool = True
+    action_disallowed_list = ["delete"]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)